src/containers/Webmap/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {FC, useEffect, useState} from 'react';
+import axios from 'axios';
+import {Icon} from 'leaflet';
+import {Map, Marker, Popup, TileLayer} from 'react-leaflet';
+import './Webmap.scss';
+import * as apiEndpoints from '../../constants/api-endpoints';
+
+export const bankIcon = new Icon({
```

Comment:
Does this need to be exported as it is not used by other components?

Reply:
Nope, doesn't need to be exported. I removed the export